src/panel/context/Timeline.tsx

```diff
@@ -41,11 +41,12 @@ export const TimelineContext = createContext<TimelineContextValue>(null as any);
 export const useTimelineContext = () => useContext(TimelineContext);
 
 const DEFAULT_WIDTH = 30000;
+export const START_PADDING = 500;
 
 const useTimelineDomain = () => {
   const startTime = useRef(Date.now());
   const domain = useRef({
-    start: startTime.current,
+    start: startTime.current - START_PADDING,
```

Comment:
Nit: it might be cool to bake the padding into these funcs as a closure? I thought about using a `Proxy` but figure that's probably a bit extra

Reply:
I agree, I'm trying to find a balance between not introducing abstraction in context (so we don't have to introduce new methods such as "setTime", "goToStart", "goToCurrent", etc) and not having this much repetition. I'll look into this soon!